### PR TITLE
REC: compatible tool for tracker hit

### PR DESCRIPTION
**Pull request by fucd**
* LCIO has TrackerHit and TrackerHitPlane, and other types, now edmp4hep uses only one type (TrackerHit), therefore to provide some functions to get covariance matrix and resolutions at U/V direction easily and uniformly
* CEPC conference description ID for sub-detectors and bit tag are also added, like LCIO

### CODE REVIEW

**Comment by fucd** [comment]
@mirguest @ihepzhangyao 

I add compatible tool for tracker hit, and then `CEPC::GetCovMatrix(edm4hep::TrackerHit& hit)` will return CovMatrix on XYZ, and `CEPC::GetResolutionRPhi(edm4hep::TrackerHit& hit)` `CEPC::GetResolutionZ(edm4hep::TrackerHit& hit)` will return resolution of RPhi or Z for any type of tracker hit. Can you take a review and give some comments?

Chengdong

**Comment by mirguest** [comment]
Hi @ihepzhangyao , can you review this PR? Thanks.
Tao

**Comment by mirguest** [comment]
Hi @fucd 
A quick comment about the Identifier. I would suggest to create a package called Identifier and move the Identifier under "Detector" directory.
Tao

File: Utilities/DataHelper/src/TrackerHitHelper.cpp [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    else if(std::bitset<32>(type)[CEPCConf::TrkHitTypeBit::PLANAR]){
+      std::array<float,6> cov{0.,0.,0.,0.,0.,0.};
+      float thetaU = hit.getCovMatrix(0);
+      float phiU   = hit.getCovMatrix(1);
+      float dU     = hit.getCovMatrix(2);
+      float thetaV = hit.getCovMatrix(3);
+      float phiV   = hit.getCovMatrix(4);
+      float dV     = hit.getCovMatrix(5);
+    
+#ifndef MethodUsedInSpacePointBuilder
```
Comment (mirguest) [inline Utilities/DataHelper/src/TrackerHitHelper.cpp:27]: Why you are using macro here? 
Tao

**Comment by fucd** [comment]
> Hi @fucd
> A quick comment about the Identifier. I would suggest to create a package called Identifier and move the Identifier under "Detector" directory.
> Tao

Hi @mirguest Sure, thank you. I will optimize the Identifier.

Chengdong

File: Utilities/DataHelper/src/TrackerHitHelper.cpp (hunk 1 above)
Comment (fucd) [reply Utilities/DataHelper/src/TrackerHitHelper.cpp:27]: @mirguest There are two comparing methods to obtain similar results. I try to choose one through macro in default environment, and leave another as reference. And them anyone can test anyone method in their environment. If macro does not recommend to use, it is also ok to remove macro and leave only one.

Chengdong

Comment (mirguest) [reply Utilities/DataHelper/src/TrackerHitHelper.cpp:27]: Hi @fucd , using macro is fine. But if you use `#ifdef`, users need to recompile the code in order to use another case. I think we need to think about it and find a better solution. 
Tao

**Review by ihepzhangyao — APPROVED** [review]

**Comment by fucd** [comment]
@ihepzhangyao 
A simple example to call:
`#include "DataHelper/TrackerHitHelper.h"`
`edm4hep::TrackerHit hit = hitVTXCol->at(ielem);`
`std::array<float,6> cov = CEPC::GetCovMatrix(hit);`
and then cov[0],cov[2],cov[5] are sigmaX^2, sigmaY^2, sigmaZ^2, respectively.

File: Utilities/DataHelper/src/TrackerHitHelper.cpp (hunk 1 above)
Comment (fucd) [reply Utilities/DataHelper/src/TrackerHitHelper.cpp]: @mirguest I moved Identifier to Detector. And I changed the method select by parameter (has a default value), is it better than macro?

**Comment by fucd** [comment]
Hi @mirguest

I found out the build crash stop at about 1m59s and repeat build at lxslc7 local. The crash is caused by missing Identifier in CMakeLists.txt while pushing. Sorry for this mistake and I have fixed it. Is it possible output the last build information while Run the build script? Currently about 112 lines are displayed.

Chengdong

**Review by mirguest — APPROVED** [review]
Looks good. Thanks.
Tao

**Pull request merged**